renderdoc/driver/d3d11/d3d11_replay.cpp

```diff
@@ -156,20 +157,30 @@ void D3D11Replay::CreateResources()
 
   RenderDoc::Inst().SetProgress(LoadProgress::DebugManagerInit, 0.9f);
 
-  AMDCounters *counters = new AMDCounters();
+  AMDCounters *pAMDCounters = new AMDCounters();
+  NVCounters *pNVCounters = new NVCounters();
```

Comment:
I know I'm kind of inconsistent about this in a lot of older written code with `m_pFoo` members but I generally don't use hungarian pointer prefix for locals like this, so `AMDCounters` and `NVCounters` would be preferred.